src/app/oiled-ramp/oiled-ramp.component.ts

```diff
@@ -20,18 +20,45 @@ export class OiledRampComponent implements OnInit, OnDestroy {
   private _destroyed$ = new Subject<void>();
   private lastAngle = 0;
   private lastWeight = 0;
+  private lastViscosity = 0;
+  private lastOilSize = 0;
+  private lastSpeed = 0;
+  private lastArea = 0;
+  private readonly gravity = 10;
```

Comment:
```suggestion
  private readonly gravity = 9.80665;
```
Use the exact number, since the studet can round it after if they want

Reply:
done